isis/scripts/downloadIsisData

```diff
@@ -110,8 +113,21 @@ def create_rclone_arguments(destination, mission_name, ntransfers=10, rclone_kwa
     if source_type == "naifKernels":
         destination = os.path.join(destination, "kernels")
 
-    extra_args=[f"{mission_name}",f"{destination}", "--progress", f"--checkers={ntransfers}", f"--transfers={ntransfers}", "--track-renames", f"--log-level={log.getLevelName(log.getLogger().getEffectiveLevel())}"]
+    # Add default filter flag
+    filter_args = ["--filter", f"{exclude_string}"]
+
+    # Check for additional include and exclude flags
+    include_args = [arg for arg in rclone_kwargs if arg.startswith("--include")]
+    exclude_args = [arg for arg in rclone_kwargs if arg.startswith("--exclude")]
 
+    # Merge any additional include and exclude flags with the default filter flag
+    if include_args:
+        filter_args[1] += "," + ",".join([arg.split("=")[1] for arg in include_args])
+    if exclude_args:
+        filter_args[1] += "," + ",".join([arg.split("=")[1] for arg in exclude_args])
```

Comment:
From the docs: https://rclone.org/filtering/

`--filter +` doesn't not import exclude `*` like include does. If the user specifies --include. If there is an --include flag, we might want to add `- *` at the bottom to simulate this? Remember, the goal is to honor the user's intention when they input their own flags, if they explicitly want the stuff we are filtering, we should be able to make that work naturally. 

Also, this doesn't seem to be accounting for the "-" or "+" in --filter to specify if it's including or excluding something? I would say read those docs I linked to get an idea of how to make this work.